src/entities/types.ts

```diff
@@ -0,0 +1,21 @@
+import { Address, Hex } from '../types';
+
+// Returned from API and used as input
+export type PoolState = {
+    id: Hex;
+    address: Address;
+    type: string;
+    tokens: TokenApi[];
+};
+
+export type TokenApi = {
+    address: Address;
+    decimals: number;
+    index: number;
+};
+
+export type Amounts = {
+    maxAmountsIn: bigint[];
+    tokenInIndex: number | undefined;
+    minimumBpt: bigint;
```

Comment:
It called my attention because I feel like I'm more used to `minBptOut`, but not a big issue, fine if we keep as is.

Reply:
I went with `minimumBpt` because it matches the encoder parameter and thought it might be best to try and follow that as much as possible this time around.